### PR TITLE
Add scaling ability to the BarModel's values axis

### PR DESCRIPTION
**Pull request by valsov**
Development for #184

### CODE REVIEW

File: OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using System.Collections.Generic;
+
+namespace OpenXMLSDK.Engine.Word.ReportEngine.Models.Charts
+{
+    /// <summary>
+    /// Class representating the scaling of a bar chart axis
+    /// </summary>
+    public class BarChartScaling
```
Comment (mathieumack) [inline OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs]: Please rename as BarChartScalingModel

File: OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    {
+        /// <summary>
+        /// Orientation of the axis
+        /// </summary>
+        public EnumValue<OrientationValues> Orientation { get; set; } = new EnumValue<OrientationValues>(OrientationValues.MinMax);
```
Comment (mathieumack) [inline OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs]: Please use your own enumeration instead Openxml enum

File: OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        /// <summary>
+        /// Maximum axis value
+        /// </summary>
+        public double? MaxAxisValue { get; set; }
+
+        /// <summary>
+        /// Construct a DocumentFormat.OpenXml.Drawing.Charts.Scaling object
+        /// </summary>
+        /// <returns></returns>
+        public Scaling GetScaling()
```
Comment (mathieumack) [inline OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs]: Please move it in Render extension.
The model doesn't include logic

File: OpenXMLSDK.Engine/Word/ReportEngine/Renders/BarChartExtensions.cs [hunk 4]
```diff
@@ -315,11 +315,7 @@ private static Run CreateBarGraph(BarModel chartModel, OpenXmlPart documentPart)
 
             // Add the Value Axis.
             dc.ValueAxis valAx = plotArea.AppendChild<dc.ValueAxis>(new dc.ValueAxis(new dc.AxisId() { Val = new UInt32Value(48672768u) },
-                new dc.Scaling(new dc.Orientation()
-                {
-                    Val = new DocumentFormat.OpenXml.EnumValue<dc.OrientationValues>(
-                        DocumentFormat.OpenXml.Drawing.Charts.OrientationValues.MinMax)
-                }),
+                chartModel.ValuesAxisScaling.GetScaling(),
```
Comment (mathieumack) [inline OpenXMLSDK.Engine/Word/ReportEngine/Renders/BarChartExtensions.cs]: Please check null value for ValuesAxisScaling field

Comment (mathieumack) [reply OpenXMLSDK.Engine/Word/ReportEngine/Renders/BarChartExtensions.cs]: If null, keep default value

Comment (valsov) [reply OpenXMLSDK.Engine/Word/ReportEngine/Renders/BarChartExtensions.cs]: Done

File: OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs (hunk 3 above)
Comment (valsov) [reply OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs]: Done

File: OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs (hunk 2 above)
Comment (valsov) [reply OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs]: Done

File: OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs (hunk 1 above)
Comment (valsov) [reply OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs]: Done

**Pull request closed without merging**